WISE-Community/WISE-Client
# feat(Authoring): Add step buttons navigate to add step route

## Changes
- Clicking on the add step buttons next to each of the steps and choosing "Add Step Before" or "Add Step After" navigates to the add step route
- Delete the "Add New Step" button at the top of the Authoring Tool
- Delete choose step location components that are no longer used

## Test
- Click the add step button to "Add Step Before" or "Add Step After"
- Make sure all four add step options work
  - Create Your Own
  - Import From Another Unit
  - Automated Assessment
  - Interactive Simulation
- Make sure going back on all the options remembers the "node id to insert inside or after" when navigating between routes
  - For example
    - Choose "Automated Assessment"
    - Choose a step
    - Click "Next"
    - Click "Back"
    - Click "Back" again
    - Now choose "Create Your Own"
    - Click "Submit"
    - This should create the new step

Closes #1708
